tests/PHPStan/Rules/Functions/ReturnTypeRuleTest.php

```diff
@@ -181,4 +181,11 @@ public function testListWithNullablesUnchecked(): void
 		$this->analyse([__DIR__ . '/data/return-list-nullables.php'], []);
 	}
 
+	public function testBug7519(): void
+	{
+		$this->checkExplicitMixed = true;
+		$this->checkNullables = true;
+		$this->analyse([__DIR__ . '/data/bug-7519.php'], []);
```

Comment:
functions vs. methods :P Please be less sloppy.

Reply:
sorry. will take a break now.